addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -174,12 +174,15 @@
                                     t-att-name="snippet.displayName"
                                     t-att-data-oe-snippet-id="snippet.id"
                                     t-att-data-module-id="snippet.moduleId"
-                                    t-on-click="_onSnippetClick"
-                                    data-tooltip="This block cannot be dropped anywhere on this page.">
+                                    t-on-click="_onSnippetClick">
                                     <t t-if="snippet.disabled">
                                         <img src="/web_editor/static/src/img/snippet_disabled.svg" class="o_snippet_undroppable"/>
                                     </t>
-                                    <div class="oe_snippet_thumbnail" t-att-class="{ 'o_we_already_dragging': snippet.renaming }" t-att-data-snippet="snippet.baseBody.dataset.snippet">
+                                    <div class="oe_snippet_thumbnail"
+                                        t-att-class="{ 'o_we_already_dragging': snippet.renaming }"
+                                        t-att-data-snippet="snippet.baseBody.dataset.snippet"
+                                        t-att-style="snippet.disabled ? 'pointer-events: auto; cursor: not-allowed;' : ''"
```

Comment:
While applying the below remark about `data-tooltip`, can you quickly triple check if there is no classes related those 2 style properties? If it's the case, it could be moved in the `t-att-class`
(Don't create it if not exist, keep it like this)